mkdocs.template.yml

```diff
@@ -115,6 +115,7 @@ nav:
       - GitHub Actions: guides/integration/github.md
       - GitLab CI/CD: guides/integration/gitlab.md
       - Pre-commit: guides/integration/pre-commit.md
+      - PyTorch: guides/integration/pytorch.md
```

Comment:
Looks like this is a duplicate, see below the fastapi line

Reply:
fixed in #9220